### PR TITLE
BaSP-TG-32: Set Up

### PR DESCRIPTION
**Pull request by ArianaMaldonado**
**Set Up description:**

> - Install DOTEnv
> - Install Jest / Supertest
> - Install mongodb-memory-server
> - Create `.env` file and move all the variables needed to that file (Port number - URL of database)
> - Create "Tests" folder into "src" and a file for each entity.
> - Create a "Seeds" folder into "src" and a file for each entity.
> - Update pre-commit  to run all test successfully before do commit (npm run test).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,33 +1,17 @@
...
-
-// Routes
-app.use(router);
-
-app.get('/', async (req, res) => {
-  res.send('Hello World!');
-});
+const MONGO_URL = process.env.DATABASE_URL;
 
 // async function adminMod()
```
Comment (lucasrendo) [inline src/index.js:11]: y esta linea?

**Review by lucasrendo — COMMENTED** [review]
- all folders are missing the files for entities other than members (i think it´s better to have them all named from the beggining to standarize and avoid team members from using different standards to name the files). 
- The folders that do contain all entities files are not all standarized names, some have the entity name in plural and others in singular, 1 of 2 conventions should be choosen, either plural or singular.
- The models entities should be in singular and uppercase

File: src/index.js (hunk 1 above)
Comment (ArianaMaldonado) [reply src/index.js:11]: Line 9? It's the constant for the URL that is used in the function of the port listener.

Comment (lucasrendo) [reply src/index.js:11]: line 22

**Review by lucasrendo — APPROVED** [review]

**Review by cocolobos — APPROVED** [review]

**Pull request merged**